bindings/nodejs/examples/client/06-simple-block.ts

```diff
@@ -22,7 +22,10 @@ async function run() {
 
     try {
         // Create block with no payload
-        const blockIdAndBlock = await client.buildAndPostBlock();
+        // TODO: have a way in the bindings to send an empty block
```

Comment:
do we have an issue for this? If we want leave TODOs as comments, then it would be nice if it also mentioned the created issue for this, otherwise this TODO may get forgotten.

Reply:
Added it, but tbh we can't forget this since this will break when we update the block and the issue is just a collection, so probably also not really useful

Reply:
okay